spec/jobs/topic_reminder_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe Jobs::TopicReminder do
+  let(:admin) { Fabricate(:admin) }
+  let(:topic) { Fabricate(:topic, topic_status_updates: [
+    Fabricate(:topic_status_update, user: admin, status_type: TopicStatusUpdate.types[:reminder])
+  ]) }
+
+  before do
+    SiteSetting.queue_jobs = true
+  end
+
+  it "should be able to create a reminder" do
+    topic
```

Comment:
This line is not necessary since we'll be initializing the record in the line below.